project/src/main/java/com/google/sps/data/JSONRestaurant.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.data;
+
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class JSONRestaurant {
```

Comment:
Lets make this class final and add a private constructor: https://screenshot.googleplex.com/f6AET3moOfV.png